firstvoices/backend/views/base_views.py

```diff
@@ -94,6 +95,8 @@ def initial(self, *args, **kwargs):
         # Finally, check permission
         perm = self.get_queryset().model.get_perm(perm_type)
         if not self.request.user.has_perm(perm, obj):
+            if perm == "backend.view_site":
```

Comment:
This condition should instead be something like "if not user.is_authenticated()"